eoscompanion/routes.py

```diff
@@ -171,6 +174,102 @@ def companion_app_server_feed_route(server, msg, path, query, *args):
     feed = dummy_feed()
     json_response(msg, feed)
 
+
+_SUFFIX_CONTENT_TYPES = {
+    '.css': 'text/css',
+    '.js': 'application/javascript'
+}
+
+
+@require_query_string_param('deviceUUID')
+@require_query_string_param('uri')
+def companion_app_server_resource_route(server, msg, path, query, *args):
+    '''Fetch an internal resource from a rewritten link on the page.
+
+    This route is for fetching internal resources not tied to any particular
+    application ID or content shard. For instance, we might embed a link
+    to a CSS or JS file that we want to serve up to the client.
+
+    The querystring param "path" indicates the URI encoded path to the
+    internal resource (it may be a GResource or a file).
+    '''
+    del args
+
+    resource_uri = Soup.URI.decode(query['uri'])
```

Comment:
The docstring above talks about the query parameter `path`, but here we use `uri` instead

Reply:
Yup, leftover. Thanks